src/commands/PublishCommand.js

```diff
@@ -380,10 +411,11 @@ export default class PublishCommand extends Command {
           const defaultVersion = semver.inc(currentVersion, "prerelease", existingId);
           const prompt = `(default: ${existingId ? `"${existingId}"` : "none"}, yielding ${defaultVersion})`;
 
+          // TODO: allow specifying prerelease identifier as CLI option to skip the prompt
```

Comment:
It seems like `--preid`, in this case, should possibly be able to provide an override (or fallback?) for the `existingId` above. It wouldn't skip the prompt, I don't think, unless they pass `--yes`.

In any case, let's leave the TODO and address it later (dunno how useful it would be, really)